projects/create-a-project.md

```diff
@@ -1,10 +1,40 @@
-# Create a Project
+# Create a Gradientº Project
 
 ## About
 
-A project is simply a directory or name for a set of related files and jobs. 
+As of 7.0, a Gradientº Project that represents a container for running Experiments.
```

Comment:
```suggestion
As of 7.0, a Gradientº Project represents a container for running Experiments.
```

Reply:
good catch. i'm actually thinking now that `container` might be a fraught term as well (in the context of docker, etc.). it felt better than `collection`, but maybe that or `group`  is more appropriate. thoughts?